frontend/src/components/layout/navigation/Navigation.module.scss

```diff
@@ -45,6 +45,8 @@
   border: none;
   cursor: pointer;
   color: currentColor;
+  // webkit adds a padding-right: 1em to button elements
+  padding: 0;
```

Comment:
This was a bonus fix, unrelated to the ticket.

Before: 
<img width="561" alt="Screen Shot 2022-10-04 at 11 00 46 PM" src="https://user-images.githubusercontent.com/2692333/193978728-5b209999-a4a6-4fa5-a02e-d08387d2e707.png">

After: 
<img width="561" alt="Screen Shot 2022-10-04 at 11 01 02 PM" src="https://user-images.githubusercontent.com/2692333/193978740-19c44f9d-0c4f-4354-897f-5152a836af73.png">

Reply:
Dammit WebKit!